chpl/chpl-resources/src/main/resources/email/chpl-email-skeleton.html

```diff
@@ -20,6 +20,11 @@
     border: 1px solid #eeeeee;
     font-family: Arial, Helvetica, sans-serif;
 }
+
+#chpltable caption {
+    caption-side:bottom;
+    text-align:left;
```

Comment:
```suggestion
    caption-side: bottom;
    text-align: left;
```

Standard format for HTML styles (at least in the UI)